src/main/java/cubicchunks/server/ChunkGc.java

```diff
@@ -97,7 +100,7 @@ private void verifyColumnConsistency() {
     }
 
     private void chunkGc() {
-        Iterator<Cube> cubeIt = cubeCache.cubesIterator();
+        Iterator<Cube> cubeIt = cubeCache.watchedCubesIterator();
```

Comment:
If you only iterate over watched cubes here, non-watched cubes will never get unloaded, which defeats the purpose of chunk GC.

Reply:
They will be unloaded if they are became watched. Imagine player surrounded by cached cubes. Inner circle - watched cubes and outer circle - synchronously loaded cubes. When player move, outer circle cubes became watched and when he leave them behind they are unloaded and removed both maps. So everything is OK.

Reply:
But that only affects those cubes which were in the direction they are moving to. As soon as you spawn there will be a sphere of non-ticking cubes around you. Unless you penetrate that sphere in every direction at least once, you will leave cubes behind.

Reply:
This is complicated a lot by the fact that player-generated cubes have `LIGHT` requirement which generates additional 2 cubes radius. As you move forwards, you leave a trail of forever-loaded cubes on the right and on the left. And any mod that accidentally loads a cube it shouldn't load will never get that cube unloaded.